gtfs_realtime_translators/registry/registry.py

```diff
@@ -12,16 +13,17 @@ class TranslatorKeyWarning(Warning):
 
 class TranslatorRegistry:
     TRANSLATORS = {
-        'la-metro': LaMetroGtfsRealtimeTranslator,
+        'la-metro-old': LaMetroGtfsRealtimeTranslator,
         'septa-regional-rail': SeptaRegionalRailTranslator,
         'septa': SwiftlyGtfsRealtimeTranslator,
         'cta-subway': CtaSubwayGtfsRealtimeTranslator,
         'cta-bus': CtaBusGtfsRealtimeTranslator,
         'mta-subway': MtaSubwayGtfsRealtimeTranslator,
         'njt-rail': NjtRailGtfsRealtimeTranslator,
         'njt-bus': NjtBusGtfsRealtimeTranslator,
-        'path': PathGtfsRealtimeTranslator,
-        'vta': SwiftlyGtfsRealtimeTranslator,
```

Comment:
@brian2000 if we delete VTA, we should delete SEPTA as well

Reply:
Done